man/udev.xml

```diff
@@ -115,7 +116,23 @@
           </listitem>
         </varlistentry>
       </variablelist>
+      </refsect2>
+
+      <refsect2><title>Values</title>
+        <para>Values are written as double quoted strings, such as <literal>"string"</literal>.
+        To include a <literal>"</literal> character in the value, use <literal>\"</literal>.
+        Any other occurrences of a character followed by a backslash are not further unescaped.
+        That is, <literal>\t\n</literal> is treated as four characters:
+        backslash, lowercase t, backslash, lowercase n.</para>
+
+        <para>The string can be prefixed with a <literal>e</literal> to mark the string as C-style escaped.
+        For example, <literal>e"string\n"</literal> is parsed as 7 characters: 6 lowercase letters and a newline.
+        This can be useful for writting special characters when a kernel driver requires them.</para>
```

Comment:
Did not enumerate all the possible escape sequences here. I'm afraid that in case escape.c is updated in the future, the table here will be outdated. Should the code be referenced directly or something better?

Reply:
Please copy the reference from man/systemd.network.xml:
`<ulink url="https://en.wikipedia.org/wiki/Escape_sequences_in_C#Table_of_escape_sequences">C-style  escapes</ulink>`